### PR TITLE
Made some changes to the web-development-resources.md file

### PR DESCRIPTION
**Pull request by alok-38**
# 🛠️ Fixes issue #
Partially updated the web-development-resources.md file

# 📇 Changes proposed 
I had proposed that the web development resource needs to be updated. Started updating post approval.

# ✅ Check List (Mark all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] This PR contains changes in the documentation
- [x] This PR contains changes in the Code of the project
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

Please have a look and suggest if it could be enhanced further.

### CODE REVIEW

File: pages/web-development-resources.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
...
+| 2. Web Development for Beginners by freeCodeCamp                        | [![Web Development](https://ik.imagekit.io/1cw2zpbjy/OSWH/FCC.png?ik-sdk-version=javascript-1.4.3&updatedAt=1670222534503 "Web Development")](https://youtu.be/nu_pCVPKzTk "Web Development")                                                                                                          | [Here you go](https://youtu.be/nu_pCVPKzTk)                                         |
+| 3. https://www.coursera.org/learn/introduction-to-front-end-development | [![Web Development](https://d3njjcbhbojbot.cloudfront.net/api/utilities/v1/imageproxy/https://coursera-course-photos.s3.amazonaws.com/fe/7320a26ec945e6ae1a3251d38d43b2/front-end-dev1.jpg "Web Development")](https://www.coursera.org/learn/introduction-to-front-end-development "Web Development") | [Here you go](https://www.coursera.org/learn/introduction-to-front-end-development) |
```
Comment (hasnainmakada-99) [inline pages/web-development-resources.md:5]: I still don't get this, the previous file was more structured than this, please rollback the changes and commit again

File: pages/AI-resource.md [hunk 2]
```diff
@@ -1 +1,7 @@
...
\ No newline at end of file
+# AI Resources
+
+## Prerequsites
+
+|         Description          |                                                                                                             Preview                                                                                                             |                                  Link to get started with it                                   |
+| :--------------------------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------: |
+| 1. Prerequisites and Prework | <center> [![Prerequisites and Prework](https://developers.google.com/static/machine-learning/images/intro-with-1_960.png "Introduction to ML And AI")](https://www.youtube.com/watch?v=BmDricQGK6w "Introduction to ML and AI") <center> | [Here you go](https://developers.google.com/machine-learning/crash-course/prereqs-and-prework) |
```
Comment (hasnainmakada-99) [inline pages/AI-resource.md:7]: Please edit the thumbnail and set the image resolution to 320wx180h, For editing the image, you can use canva or any other tools

**Review by hasnainmakada-99 — CHANGES_REQUESTED** [review]

**Comment by hasnainmakada-99** [comment]
@alok-38 , please do the changes I requested

**Comment by alok-38** [comment]
Okay

**Pull request closed without merging**